gradle.properties

```diff
@@ -2,7 +2,7 @@
 # -> https://www.jetbrains.org/intellij/sdk/docs/reference_guide/intellij_artifacts.html
 pluginGroup=com.github.intheclouddan.intellijpluginld
 pluginName=intellij-plugin-ld
-pluginVersion=0.5.1-idea22
+pluginVersion=1.0.0-idea21
```

Comment:
realized we should have changed this to `-idea21` for 0.5.1 when we switched compatibility range